shared/react-native/ios/fastlane/Fastfile

```diff
@@ -25,9 +25,10 @@ platform :ios do
   desc "Submit a new Beta Build to Apple TestFlight"
   desc "This will also make sure the profile is up to date"
   lane :beta do
-  version = get_version_number
+  version = get_version_number(xcodeproj: ENV['XCODEPROJ_NAME'], target: 'Keybase')
     increment_build_number({
-      build_number: latest_testflight_build_number(version: version) + 7
+      build_number: latest_testflight_build_number(version: version) + 3,
+      target: "Keybase"
```

Comment:
i think this break things: `You passed invalid parameters to 'increment_build_number'`

Reply:
this is a wip